src/components/slider/slider.js

```diff
@@ -643,15 +643,15 @@ function SliderDirective($$rAF, $window, $mdAria, $mdUtil, $mdConstant, $mdThemi
      * @returns {number}
      */
     function positionToPercent(position) {
-      var offset = vertical ? sliderDimensions.top : sliderDimensions.left;
+      var offset = vertical ? (sliderDimensions.top + window.scrollY) : (sliderDimensions.left + window.scrollX);
```

Comment:
So the idea is that this makes the offset relative to the current viewport (that is scrolled into view) rather than the entire page?

Reply:
@Splaktar No it's opposite. `sliderDimensions.top` and `sliderDimensions.left` is actually relative to current viewport. So we need to `+ scrollY` to make it become relative to the entire page instead. And it would match with `pointer.x`